### PR TITLE
Add UserGuide

### PR DESCRIPTION
**Pull request by jamesebond**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -308,7 +314,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use Case: View All Stalls**
```
Comment (teoks0199) [inline docs/DeveloperGuide.md:317]: Clear use case

**Review by teoks0199 — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -263,22 +263,28 @@ _{Explain here how the data archiving feature will be implemented}_
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: User-friendly interface makes recording and retrieving information a breeze.
```
Comment (Ruizhi2001) [inline docs/DeveloperGuide.md:266]: clear value proposition

**Review by Ruizhi2001 — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -263,22 +263,28 @@ _{Explain here how the data archiving feature will be implemented}_
...
+Convenient food diary that records your restaurant visits, menu items.
+No more forgetting the name of that delightful dish you had last week; 
+it's all neatly stored in your digital food diary. Able to sort by prices, and ratings, good for students
+on a budget!
```
Comment (samuelmui8) [inline docs/DeveloperGuide.md:270]: Nice value proposition, maybe can include CLI also cuz its a requirement

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (samuelmui8) [inline docs/DeveloperGuide.md:317]: Use case is clear

**Review by samuelmui8 — APPROVED** [review]
LGTM

**Pull request closed without merging**